### PR TITLE
Chips badge

### PR DESCRIPTION
**Pull request by AnkurChunekar**
- chips added.
- badges  added
- fixed navigation flex class

screen-recorder-thu-feb-03-2022-20-16-06
screen-recorder-thu-feb-03-2022-20-15-29

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
pr good to merge

File: components/chips/chips.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    const div = document.createElement("div");
+    div.setAttribute("class", "chip-body black-chip")
+    const span = document.createElement("span");
+    span.innerText = chipLabel;
+    const i = document.createElement("i");
+    i.setAttribute("class", "fas fa-times close");
+
+    div.appendChild(span);
+    div.appendChild(i);
```
Comment (VishalPatil18) [inline components/chips/chips.js:16]: This is a good way, but we could also do something like this which does the work in fewer lines and also we don't have to worry about the hassle of setAttribute and createElement and all that stuff...

```javascript
const chipElement = 
    `
        <div class="chip-body black-chip">
             <span>chipLabel</span>
             <i class="fas fa-times close"></i>
        </div> 
    `;
return chipElement;
```

**Review by VishalPatil18 — APPROVED** [review]
Just one suggestion was added. Apart from that the code looks good and works great! Great stuff @AnkurChunekar 👍

File: components/chips/chips.js (hunk 1 above)
Comment (AnkurChunekar) [reply components/chips/chips.js:16]: Hey, Vishal thanks a lot for you feedback, will look into this.

Comment (AnkurChunekar) [reply components/chips/chips.js:16]: Hey Vishal if we try to implement it in this way then i guess we have to use innerText later, which i have read somewhere using it is a bad practice. I did it in this way first, then again i changed it to the createElement and setattribute one.

Comment (VishalPatil18) [reply components/chips/chips.js:16]: You might be but I also did in a very similar way as I mentioned and there I used `innerHTML`. You can check [this](https://codepen.io/VishalPatil18/pen/poWXpez?editors=0010) for reference.

Comment (AnkurChunekar) [reply components/chips/chips.js:16]: Thanks @VishalPatil18 will look into it.

**Pull request merged**